docs/radicle.md

```diff
@@ -20,19 +20,26 @@ For more on Radicle Link, check out the [Understanding Radicle](understanding-ra
 
 ## How is Radicle different from GitHub?
 
-While GitHub and other platforms offer an easy and standard way to collaborate on code, their centrally-hosted approach has surfaced numerous platform and security risks. Despite relying on the open protocols of Git, these platforms are vulnerable to censorship as well as capitalist ends, which are often misaligned with the goals of free and open source software.
+When coming to Radicle from a centralized code collaboration network like Github or Gitlab, you might notice that Radicle priotizes different concerns. Concretely:
 
-As the status quo for code collaboration, these platforms host the largest repositories of open source development made up of not just code, but issues, pull requests, reviews, and comments. These parts of the code collaboration process exist solely within these platforms, along with the largest social network for developers in the world. 
+1. users being in control of their identity
+2. users being in control of their content
+3. users being in control of their social interactions
 
-Radicle was designed to be a robust, functional, and secure alternative for code collaboration that anyone in the world can use, however they want, whenever they want.
+These concerns lead to a model for collaboration that might be familiar to the free and open source hackers of the 90s and early 2000s, that is radically different to the one presented by the web 2.0 platforms.
 
-[Read more about our vision of decentralized code collaboration.](understanding-radicle/why-radicle.md)
+The radicle-link protocol distinguishes between two types of identities: personal and project. The first describes an actor in the system, while the second describes a (software) project on which one or more actors collaborate. In radicle:
 
-The goal is to provide everything that a centrally-hosted platform does without relying on intermediaries or third parties. With this in mind, Radicle's peer-to-peer approach presents a new model for code collaboration in which the idea of a single canonical upstream is replaced by bazaar-style development. 
+1. peers follow other peers.
+2. peers track projects they are interested in.
+3. peers gossip about projects. This means tracking and replicating objects from the peers they follow, about the projects they are interested in.
+
+It's this "trusted" social graph of peers and projects that becomes the foundation for collaboration within Radicle.
+
+If you have already interacted with other decentralized social networks, this model might remind you of secure scuttlebutt. In fact, Radicle is heavily inspired by secure scuttlebutt's design but diverges from SSB in a few ways.
```

Comment:
Tbh I don't think we should include this here... Let's definitely include it in the FAQ, but no need to push readers away to another project's website.